CMakeLists.txt

```diff
@@ -666,125 +666,99 @@ if(BUILD_TESTS)
+    PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/e2e_scenarios.py
+    CONSENSUS raft
+    ADDITIONAL_ARGS --scenario
+                    ${CMAKE_SOURCE_DIR}/tests/simple_logging_scenario.json
+  )
 
-    add_e2e_test(
-      NAME ws_scaffold_${CONSENSUS}
-      PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/ws_scaffold.py
-      CONSENSUS ${CONSENSUS}
-    )
+  add_e2e_test(
+    NAME ws_scaffold_raft
+    PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/ws_scaffold.py
+    CONSENSUS raft
+  )
 
-    add_e2e_test(
-      NAME lua_e2e_logging_${CONSENSUS}
-      PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/e2e_logging.py
-      CONSENSUS ${CONSENSUS}
-      ADDITIONAL_ARGS --app-script
-                      ${CMAKE_SOURCE_DIR}/src/apps/logging/logging.lua
-    )
+  add_e2e_test(
+    NAME lua_e2e_logging_raft
+    PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/e2e_logging.py
+    CONSENSUS raft
+    ADDITIONAL_ARGS --app-script
+                    ${CMAKE_SOURCE_DIR}/src/apps/logging/logging.lua
+  )
+
+  add_e2e_test(
+    NAME member_client_test_raft
+    PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/memberclient.py
+    CONSENSUS raft
+  )
 
+  if(NOT SAN)
     add_e2e_test(
-      NAME member_client_test_${CONSENSUS}
-      PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/memberclient.py
-      CONSENSUS ${CONSENSUS}
+      NAME connections_raft
+      PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/connections.py
+      CONSENSUS raft
     )
+  endif()
 
-    if(NOT SAN)
-      add_e2e_test(
-        NAME connections_${CONSENSUS}
-        PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/connections.py
-        CONSENSUS ${CONSENSUS}
-      )
-    endif()
+  add_e2e_test(
+    NAME receipts_test_raft
+    PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/receipts.py
+    CONSENSUS raft
+  )
 
+  if(TLS_TEST)
     add_e2e_test(
-      NAME receipts_test_${CONSENSUS}
-      PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/receipts.py
-      CONSENSUS ${CONSENSUS}
+      NAME tlstest_raft
+      PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/tlstest.py
+      CONSENSUS raft
+      LABEL tlstest
     )
+  endif()
 
-    if(TLS_TEST AND ${CONSENSUS} STREQUAL raft)
-      add_e2e_test(
-        NAME tlstest_${CONSENSUS}
-        PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/tlstest.py
-        CONSENSUS ${CONSENSUS}
-        LABEL tlstest
-      )
-    endif()
+  add_e2e_test(
+    NAME schema_test_raft
+    PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/schema.py
+    CONSENSUS raft
+    ADDITIONAL_ARGS -p liblogging --schema-dir ${CMAKE_SOURCE_DIR}/doc/schemas
+  )
 
-    add_e2e_test(
-      NAME schema_test_${CONSENSUS}
-      PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/schema.py
-      CONSENSUS ${CONSENSUS}
-      ADDITIONAL_ARGS -p liblogging --schema-dir
-                      ${CMAKE_SOURCE_DIR}/doc/schemas
-    )
+  add_e2e_test(
+    NAME election_test_raft
+    PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/election.py
+    CONSENSUS raft
+    ADDITIONAL_ARGS "--raft-election-timeout" "4000"
+  )
 
+  foreach(CONSENSUS ${CONSENSUSES})
```

Comment:
I'm a little confused by this change, is the purpose to have one test (the cpp logging) running with a bft config?